config.mk

```diff
@@ -45,6 +45,11 @@ LIBS += -L${MSTWPDF_HOME} -lmstwpdf
 DEPS += -I${ADAGE_HOME}/include
 LIBS += -L${ADAGE_HOME}/lib -lAdage
 
+#PYBIND
+DEPS += -I/usr/._local/4lciz2i3o3tp525sudm44xu6vhqszeil/include/python3.10 -I/usr/._local/4lciz2i3o3tp525sudm44xu6vhqszeil/include/python3.10
+LIBS += -L/usr/._local/4lciz2i3o3tp525sudm44xu6vhqszeil/lib -lcrypto -lintl -ldl  -lm -lm -lpython3.10
+DEPS += -I/w/hallb-scshelf2102/clas12/pecar/eic/sidis-eic/deps/pybind11/include
```

Comment:
we might not need the `DEPS` additions in `eic-shell`; both `python3.10` and `pybind11` are in `/usr/local/include`